src/java.base/windows/classes/sun/nio/fs/WindowsPathParser.java

```diff
@@ -147,6 +166,12 @@ private static Result parse(String input, boolean requireToNormalize) {
             }
         }
 
+        if (prefixType == PREFIX_TYPE_LONG && type != WindowsPathType.ABSOLUTE) {
+            throw new InvalidPathException(input, "Long path prefix must precede an absolute path");
+        } else if (prefixType == PREFIX_TYPE_LONG_UNC && type != WindowsPathType.UNC) {
+            throw new InvalidPathException(input, "Long UNC path prefix must precede a UNC path");
```

Comment:
I don't think the text should have "must precede" as someone reading this might assume they must use this prefix. Maybe say "cannot be used with ..." ?

Reply:
Fixed in 90ef2d30a9ed8f7ddb5891836663e5f8728a948d.